release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -2828,3 +2828,26 @@ link:https://access.redhat.com/solutions/6957131[{product-title} 4.9.32 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[[ocp-4-9-33]]
+=== RHSA-2022:2205 - {product-title} 4.9.33 bug fix and security update
+
+Issued: 2022-05-17
+
+{product-title} release 4.9.33, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:2206[RHBA-2022:2206] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:2205[RHSA-2022:2205] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6958609[{product-title} 4.9.33 container image list]
+
+[id="ocp-4-9-33-bug-fixes"]
+==== Bug fixes
+* Previously, {product-title} 4.8 added an API for customizing the console and OAuth platform routes. This API included the `status` field to report the host names of customizable routes and the `spec` field to report the user host names in the cluster ingress configuration.  The API did not include the host names with top-level domains (TLDs) that contained decimal digits. Consequently, the users failed to specify the custom host names in the cluster ingress configuration and install the clusters with TLDs containing the decimal digits. With this update, the users can include the TLDs containing the decimal digits. As a result, {product-title} 4.8 allows users to use the cluster domains with TLDs containing the decimal digits. Additionally, {product-title} 4.8 allows the users to customize the console and OAuth routes by using the host names that are valid for routes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2075551[*BZ#2075551*])
+
+* Previously, to configure DNS on the Azure using the installer-provisioned infrastructure (IPI) the ingress operator required an Azure
+Resource Manager (ARM) endpoint. This endpoint needed to be provided through Azure Stack Hub configuration. Consequently, the ingress operator failed to configure a wildcard DNS record for ingress on Azure Stack Hub. With this update, the ingress operator can check the cluster infrastructure config object for the ARM endpoint provided by the installer. As a result, the ingress operator uses the ARM endpoint to configure the wildcard DNS record on the Azure Stack Hub. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2032677[*BZ#2032677*])
```

Comment:
Is cluster infrastructure config object an official name? Normally, map always follows config unless it is an official name.

Reply:
I don't think it's an official name. I changed the `config` to `configuration`.